static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -59,11 +62,16 @@ const init = (start: () => void, stop: () => void): Promise<void> => {
             // A promise error here, from a failed module load,
             // could be a network problem or an intercepted request.
             // Abandon the init sequence.
-            .catch(removeAdSlots);
+            .catch(forceRemoveAdSlots);
+        return Promise.resolve();
+    }
+
+    if (commercialFeatures.adFree) {
+        setupAdvertising().then(removeAdSlots).catch(forceRemoveAdSlots);
```

Comment:
`removeAdSlots` is sensitive to `adFree`. We retain some slots for merchandising etc. `forceRemoveAdSlots` removes everything.